core/lib/spree/testing_support/factories/address_factory.rb

```diff
@@ -11,7 +11,7 @@
     end
 
     firstname 'John'
-    lastname 'Doe'
+    lastname nil
```

Comment:
> In general we want factories not to set attributes that aren't actually required

Is this true? I think of factories as creating 'typical' objects, not neccessarily minimal objects.